### ISSUE TITLE
Request cold-start task code

### ISSUE THREAD

**Issue by Ngqgq** [guaguabujianle/SA-DDI#1]
Hello, sorry to bother you. I'm interested in your work and have run through successfully, but there is only the code for the warm start task in the home page, Colud you send me the two task codes for the cold start？

**Comment by guaguabujianle** [comment]
You can contact me via 418951722@qq.com. I will send you some relevant codes.

**Comment by Woiea** [comment]
Hello, I would like to obtain the relevant code for the cold-start task (including the code for obtaining cold-start task data), and I have sent a request to the email address you provided. Looking forward to your reply.

**Comment by YoungbinCho** [comment]
Hello, I was really impressed by your research. I have a question about your experiment of cold start setting too.
I made the cold start dataset by myself and train this model, and I encountered an overfitting issue.
So I'm curious if you've experienced the same issue. And if you didn't, I want to ask you the cold start dataset you made.
I would greatly appreciate any insights or advice you could provide. Thank you in advance for your time and help, and I look forward to hearing from you

**Comment by guaguabujianle** [comment]
> Hello, I was really impressed by your research. I have a question about your experiment of cold start setting too. I made the cold start dataset by myself and train this model, and I encountered an overfitting issue. So I'm curious if you've experienced the same issue. And if you didn't, I want to ask you the cold start dataset you made. I would greatly appreciate any insights or advice you could provide. Thank you in advance for your time and help, and I look forward to hearing from you

Thank you for your kind words about my research. Regarding your question about the cold start setting, overfitting is indeed a common issue in such scenarios. In my experience, applying strong regularization techniques—such as dropout and weight decay—has proven to be effective in mitigating this issue. These methods are also recommended in the readme file along with this GitHub repository.

**Comment by YoungbinCho** [comment]
I missed that there is a section on overfitting in your paper. As you said, modifying the weight decay has helped with the learning process. Thank you for your quick response.
